src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/HearingFeeDueHandler.java

```diff
@@ -49,6 +49,7 @@ public void handleTask(ExternalTask externalTask) {
                     applicationEventPublisher.publishEvent(new HearingFeeUnpaidEvent(caseDetails.getId()));
                 }
             } catch (Exception e) {
+                //log exception and continue with other cases
```

Comment:
As I'm the one who asked to put these comments:
what will happen in terms of logic in the code is fairly obvious. The comment should make it clear that this is on purpose.

I would suggest rephrasing the comment as something on the line of:
`//we continue handling other cases on purpose, as we don't want to stop processing of valid cases just because an error happened on one. We log the error to leave a trace that something may need to be looked into`

Reply:
done